redfish-core/lib/log_services.hpp

```diff
@@ -51,6 +53,24 @@ constexpr char const* crashdumpOnDemandInterface =
 constexpr char const* crashdumpTelemetryInterface =
     "com.intel.crashdump.Telemetry";
 
+constexpr std::array<const char*, 3> hwIsolationEntryIfaces = {
+    "xyz.openbmc_project.HardwareIsolation.Entry",
+    "xyz.openbmc_project.Association.Definitions",
+    "xyz.openbmc_project.Time.EpochTime"};
+
+using RedfishUriListType =
+    std::vector<std::tuple<std::vector<std::string>, std::string>>;
+
+const RedfishUriListType redfishUriList = {
```

Comment:
I think you are better off coding this here and just containing this to this file..

Reply:
I think as is (e.g. processorInterfaces.begin(), processorInterfaces.end()), ) makes this harder to understand. 
I think just hardcoding these makes this easier to carry downstream (touches less files)
I don't think these will change in the future and if they do this code might need to be adjusted anyway